modules/central-configuration-options-operator.adoc

```diff
@@ -40,6 +40,9 @@ Otherwise, you must reinstall the custom resource to log back in.
 |`central.exposure.route.enabled`
 |Set this to `true` to expose Central through an OpenShift route. The default value is `false`.
 
+|`central.exposure.route.host`
+|Specify a custom host name to use for Central's route. Leave this blank to accept the default value from OpenShift.
```

Comment:
```suggestion
|Specify a custom host name to use for Central's route. Leave this unset to accept the default value from OpenShift.
```
Not sure how a route would behave if host is set to an empty string (as opposed to unset), but better safe than sorry.